metrics/storage/webtooling.sh

```diff
@@ -13,8 +13,13 @@ set -o pipefail
 # General env
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../lib/common.bash"
+NUM_CONTAINERS="$1"
+
+# runtimes: io.containerd.runc.v2, io.containerd.run.kata.v2
+CTR_RUNTIME="${CTR_RUNTIME:-io.containerd.run.kata.v2}"
```

Comment:
you do not need to specify again the value of `CTR_RUNTIME` as it is already defined at the common.bash

Reply:
got it, thank you @GabyCT